common/src/main/scala/org/apache/celeborn/common/CelebornConf.scala

```diff
@@ -3213,6 +3214,15 @@ object CelebornConf extends Logging {
       .timeConf(TimeUnit.MILLISECONDS)
       .createWithDefaultString("600s")
 
+  val CLIENT_FETCH_BUFFER_MAX_SIZE: ConfigEntry[Long] =
+    buildConf("celeborn.client.fetch.buffer.max.size")
+      .withAlternative("celeborn.fetch.buffer.max.size")
```

Comment:
why add this alternative?

Reply:
Copied from `celeborn.client.push.buffer.max.size`, is it necessary?

Reply:
understand what the code does before copying. it is for backward compatibility so that alternatives will also work, which is obviously not required here.

Reply:
Got it, thanks for the reminder.